app/src/main/java/io/github/pietrocaselani/moviestraker/di/AppInjector.kt

```diff
@@ -0,0 +1,59 @@
+package io.github.pietrocaselani.moviestraker.di
+
+import android.app.Activity
+import android.os.Bundle
+import android.support.v4.app.Fragment
+import android.support.v4.app.FragmentActivity
+import android.support.v4.app.FragmentManager
+import android.support.v4.app.FragmentManager.FragmentLifecycleCallbacks
+import dagger.android.AndroidInjection
+import dagger.android.support.AndroidSupportInjection
+import dagger.android.support.HasSupportFragmentInjector
+import io.github.pietrocaselani.moviestraker.ActivityLifecycleCallbacksAdapter
+import io.github.pietrocaselani.moviestraker.MoviesTrackerApplication
+import io.github.pietrocaselani.moviestraker.ui.moviedetails.MovieDetailsModule
+import io.github.pietrocaselani.moviestraker.ui.searchmovies.SearchMoviesModule
+import io.github.pietrocaselani.moviestraker.ui.upcoming.UpcomingModule
+
+/**
+ * Created by pc on 24/06/17.
+ */
+object AppInjector {
```

Comment:
This is a fancy way of injecting stuff :-) 

Can you explain us why you chose to do this way?